python/lsst/ts/m2com/controller.py

```diff
@@ -81,7 +88,14 @@ def __init__(self, log=None, timeout_in_second=0.05, maxsize_queue=1000):
 
         self.timeout = timeout_in_second
 
-        self.controller_state = salobj.State.OFFLINE
+        # Is CSC or not. Remove this after the state machines in cell
```

Comment:
I have no ticket numbers because I will need to rewrite a lot of the M2 cell LabVIEW code for this (which will take me several months). Another option is to let John (in SLAC) to fix this part in M2 cell C++ code directly. I do not decide yet and will need to discuss with John. Thanks!